examples/oembed.html

```diff
@@ -13,22 +13,27 @@
 
 <div class="mobile-alerts">Loading...</div>
 
-<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-<script src="//netdna.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
-<script src="http://api.alerta.io/embed.js"></script>
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<script src="http://netdna.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
+<script src="http://localhost:8080/api/embed.js"></script>
```

Comment:
Using the `/api` path prefix here is not correct because it will work only in your environment. Please, use more general URL like this: `//localhost:8080/embed.js`

Reply:
Fixed.